services/static-webserver/client/source/class/osparc/Application.js

```diff
@@ -421,8 +421,13 @@ qx.Class.define("osparc.Application", {
     /**
      * Resets session and restarts
     */
-    logout: function() {
-      osparc.FlashMessenger.getInstance().logAs(this.tr("You are logged out"));
+    logout: function(forcedReason) {
+      if (forcedReason) {
+        const msg = this.tr("You were logged out:") + "<br>" + forcedReason;
+        osparc.FlashMessenger.getInstance().logAs(msg, "WARNING", 0);
+      } else {
+        osparc.FlashMessenger.getInstance().logAs(this.tr("You are logged out"), "INFO");
```

Comment:
You **were** logged out? or simply **Logged out** Double check spelling :-)

Reply:
Since it was the platform logging the user out, I thought it was better to go say "You were logged out". The other message is shown when the user voluntarily logs out.